MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -383,13 +451,15 @@ + (void)setAutomaticVideoChange:(NSString *)name  enabled:(Boolean)enabled {
 
 #pragma mark Update Customer Data
 
-+ (void)updateCustomerDataForPlayer:(nonnull NSString *)name withPlayerData:(nullable MUXSDKCustomerPlayerData *)playerData withVideoData:(nullable MUXSDKCustomerVideoData *)videoData {
-    [self updateCustomerDataForPlayer:name withPlayerData:playerData withVideoData:videoData viewData:nil];
-}
-
-+ (void)updateCustomerDataForPlayer:(nonnull NSString *)name withPlayerData:(nullable MUXSDKCustomerPlayerData *)playerData withVideoData:(nullable MUXSDKCustomerVideoData *)videoData viewData: (nullable MUXSDKCustomerViewData *) viewData {
++ (void)setCustomerData:(nullable MUXSDKCustomerData *)customerData forPlayer:(nonnull NSString *)name {
     MUXSDKPlayerBinding *player = [_viewControllers valueForKey:name];
     if (!player) return;
+
+    MUXSDKCustomerPlayerData *playerData = [customerData customerPlayerData];
+    MUXSDKCustomerViewData *viewData = [customerData customerViewData];
+    MUXSDKCustomerVideoData *videoData = [customerData customerVideoData];
+    _customerViewerData = [customerData customerViewerData]; // call initSDK?
```

Comment:
this should _probably_ call initSDK again, or any changes to customerViewerData don't get packaged into an event until something else calls initSDK

Reply:
alternatively, this could call out to `dispatchGlobalDataEvent` right?

Reply:
Perhaps this piece in `initSDK` specifically is worth putting into a separate method so we can do that
```
MUXSDKViewerData *viewerData = [[MUXSDKViewerData alloc] init];
    NSString *bundleId = [[NSBundle mainBundle] bundleIdentifier];
    if (bundleId) {
        [viewerData setViewerApplicationName:bundleId];

    NSString *viewerApplicationName = [_customerViewerData viewerApplicationName];
    if (viewerApplicationName == nil) {
        viewerApplicationName = [[NSBundle mainBundle] bundleIdentifier];
    }
    [viewerData setViewerApplicationName:viewerApplicationName];
 ```

Reply:
yes that is an option as well